Resources/Prototypes/_DV/Loadouts/loadout_groups.yml

```
@@ -770,6 +778,14 @@
   - WhiteShoes
   - ScienceWinterBoots
 
+- type: loadoutGroup
+  id: RoboticistGlasses
+  name: loadout-group-roboticist-glasses
+  minLimit: 0
+  loadouts:
+  - ClothingEyesHudDiagnostic
+  - ClothingEyesPrescriptionDiagHud
```

Add `ClothingEyesEyepatchHudDiag` as well.


```suggestion
  loadouts:
  - ClothingEyesHudDiagnostic
  - ClothingEyesPrescriptionDiagHud
  - ClothingEyesEyepatchHudDiag
```